Userland/Libraries/LibC/malloc.cpp

```diff
@@ -153,12 +155,66 @@ static inline BigAllocator (&big_allocators())[1]
     return reinterpret_cast<BigAllocator(&)[1]>(g_big_allocators_storage);
 }
 
-static Allocator* allocator_for_size(size_t size, size_t& good_size)
+// --- BEGIN MATH ---
+// This stuff is only used for checking if there exists an aligned block in a
+// chunk. It has no bearing on the rest of the allocator, especially for
+// regular malloc.
+
+static inline unsigned long modulo(long a, long b)
+{
+    return (b + (a % b)) % b;
+}
+
+// Returns x, y, gcd.
+static inline Tuple<long, long, long> extended_euclid(long a, long b)
```

Comment:
Please just make a struct, AK/Tuple isn't meant to be used in normal code as a replacement for structs

Reply:
done